kerckhoff/packages/models.py

```diff
@@ -0,0 +1,57 @@
+import uuid
+
+from django.contrib.auth.models import User
+from django.contrib.postgres.fields import JSONField
+from django.db import models
+
+from .constants import *
+
+class PackageSet(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    slug = models.SlugField(max_length=32, unique=True) # ??
+    drive_folder_id = models.CharField(max_length=512, blank=True)
+    drive_folder_url = models.URLField()
+    metadata = JSONField(blank=True, default=dict, null=True)
+    # created_by = models.ForeignKey(User, on_delete=models.CASCADE)
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+    
+class Package(models.Model):
+    # slug = models.CharField(max_length=64, primary_key=True) # ??
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    slug = models.CharField(max_length=64)
+    package_set = models.ForeignKey(PackageSet, on_delete=models.PROTECT)
+    
+    class Meta:
+        unique_together = ('package_set', 'slug',)
+
+    # description = models.TextField(blank=True)
+    drive_folder_id = models.CharField(max_length=512)
```

Comment:
remove drive_folder_id, drive_folder_url, (we'll put those in metadata), images, data, processing, publish_date, _content_type, add created_by, and rename cached_article_preview to just cached, and change it to a JSONField